rpc/documentation/api.md

```diff
@@ -792,6 +838,137 @@ wallet and republished later if it or a double spend are not mined.
 
 ___
 
+#### `TicketPrice`
+
+The `TicketPrice` method returns the price of a ticket for the next block, also 
+known as the stake difficulty. May be incorrect if the daemon is currently 
+syncing.
+
+**Request:** `TicketPriceRequest`
+
+**Response:** `TicketPriceResponse`
+
+- `int64 ticket_price`: The stake difficulty for the next block.
+
+**Expected errors:**
+
+- `Unknown`: The daemon was not connected to the wallet.
```

Comment:
Unknown is the opposite of an expected error, it is the fallback value when we are not converting an expected error to a usable status code.